### DISCUSSION TITLE
It's possible to embed HTML5 video in embed compontent?

### DISCUSSION THREAD

**Discussion by rflstrmn** [Budibase/budibase#8474] [Help]
Hello Buddibase community,

I'm trying to get a video with the "embed" component, in on a screen. It shows me the player, but no video. Anyone have an idea what this could be?
`<video controls poster="http://www.example.com/waterfall.png" preload="metadata">
    <source src="http://www.example.com/waterfall-video.mp4" type="video/mp4"/>
    <source src="http://www.example.com/waterfall-video.webm" type="video/webm"/>
    <source src="http://www.example.com/waterfall-video.ogv" type="video/ogg"/>
    Video not supported.
</video>`

**Comment by andz-bb** [comment]
hey @rflstrmn, in this case the example videos don't exist there on example.com, but the video element works fine within an embed component when you replace those with valid URLs.

Here's one I used in an embed that works using an example URL hosting a video.

```
<video controls poster="https://www.sample-videos.com/img/Sample-png-image-500kb.png" preload="metadata"> <source src="http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ElephantsDream.mp4" type="video/mp4"/> Video not supported. </video>
```

Hope this helps!
